src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -1349,9 +1349,13 @@ public Vector2 ScreenPosition
             get
             {
                 Vector2 temp = new Vector2(0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
```

Comment:
thank you for the patch.
but could you add `private Vector2 internalScreenPosition` as View's member and use it.
and in View's Dispose(), make it disposed.
could you please refer to these? 
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/View.cs#L89
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs#L1526
thank you.

Reply:
Fix done. I make `internalCurrentScreenPosition` and get in on `ViewInternal.cs`.